Sources/System/Errno.swift

```diff
@@ -1361,7 +1361,7 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   @available(*, unavailable, renamed: "protocolError")
   public static var EPROTO: Errno { protocolError }
 
-#if !os(OpenBSD) && !os(WASI)
+#if !os(FreeBSD) && !os(OpenBSD) && !os(WASI)
```

Comment:
```suggestion
#if !os(OpenBSD) && !os(WASI) && !os(FreeBSD)
```
Just to keep the order consistent